components/mpas-framework/src/tools/registry/gen_inc.c

```diff
@@ -1407,6 +1434,86 @@ int parse_var_array(FILE *fd, ezxml_t registry, ezxml_t superStruct, ezxml_t var
 				fortprintf(fd, "         call mpas_add_att(%s %% attLists(const_index) %% attList, 'units', '%s')\n", pointer_name_arr, temp_str);
 			}
 
+			if ( varbounds != NULL ) {
+				string = strdup(varbounds);
+				tofree = string;
+
+				token = strsep(&string, "'");
+				sprintf(temp_str, "%s", token);
+
+				while ( ( token = strsep(&string, "'") ) != NULL ) {
+					sprintf(temp_str, "%s''%s", temp_str, token);
+				}
+
+				free(tofree);
+
+				fortprintf(fd, "         call mpas_add_att(%s %% attLists(const_index) %% attList, 'bounds', '%s')\n", pointer_name_arr, temp_str);
+			}
```

Comment:
I tried adding attributes to a `var_array` and they didn't come through in the code.  I added:
```xml
                <var_struct name="tracers" time_levs="2">
-                       <var_array name="activeTracers" dimensions="nVertLevels nCells Time" type="real" packages="activeTracersPKG" >
+                       <var_array name="activeTracers" dimensions="nVertLevels nCells Time" type="real" packages="activeTracersPKG"
+                                       coordinates="lonCell latCell"
+                                       bounds="lonVertex"
+                                       cell_method="area: mean where sea"
+                                       cell_measures="area: areaCell volume: volCell">
                                <var name="temperature" array_group="activeGRP" units="degrees Celsius"
+                                       standard_name="sea_water_potential_temperature"
                         description="potential temperature"
                                />
                                <var name="salinity" array_group="activeGRP" units="grams salt per kilogram seawater"
+                                       standard_name="sea_water_salinity"
                         description="salinity"
                                />
                        </var_array>

```
And the `standard_name` came through for each variable but the attributes on the `var_array` are absent in `src/inc/structs_and_variables.inc`.